internal/indexer/indexer.go

```diff
@@ -391,13 +391,17 @@ func (i *Indexer) indexReferences() {
 
 // indexReferencesForPackage emits data for each reference within the given package.
 func (i *Indexer) indexReferencesForPackage(p *packages.Package) {
-	for ident, obj := range p.TypesInfo.Uses {
-		pos, d, ok := i.positionAndDocument(p, ident, obj)
+	for ident, definitionObj := range p.TypesInfo.Uses {
+		if definitionObj == nil {
+			continue
+		}
```

Comment:
question:
just for understanding `golang.org/x/tools/go/packages`'s sake, in what cases does this happen?

Reply:
I don't know - I may have seen it on `TypeInfo.Uses` only but I'd rather be safe here since I don't know why it would be nil in either of these places 😬

Reply:
I'll look into it though and post here if I'm enlightened.

Reply:
For `TypeInfo.Defs`:

> For identifiers that do not denote objects (e.g., the package name in package clauses, or symbolic variables t in t := x.(type) of type switch headers), the corresponding objects are nil.